### ISSUE TITLE
Can't find how to launch the osk

### ISSUE THREAD

**Issue by wjbite** [MirServer/ubuntu-frame-osk#43]
Ubuntu 22.04 installed this package  through UBUNTU SOFTWARE and it seemed to install okay.
However, I can't figure out how to launch the program.
It doesn't appear in SHOW APPLICATIONS or in the task bar.
Thanks,
Walt Bankes

**Comment by AlanGriffiths** [comment]
Hi Walt,

I'm not sure why you expect the OSK to be launched like an end-user application. There's documentation for it here:

https://discourse.ubuntu.com/t/ubuntu-frame-osk-documentation/27568

(That ought to be easier to find, so I'll leave the issue open until we can add that to our documentation backlog.)

**Comment by wjbite** [comment]
Thanks Alan for your quick reply.
My bad! I did not realize that this was not an end-user application. I have been under the impression that all of the packages in the UBUNTU SOFTWARE folder installed by default with Ubuntu desktop are end-user apps - not so!
A Ukrainian friend who uses Windows needed to send email in that language. I searched and found Windows OSK and the language package - installed it and, VIOLA. Then I thought that maybe I had that available with Ubuntu. Maybe someday.
Obviously, I am not a developer :(
Thanks for your help.
